src/domain/resource/resourceKeyboardReservation/ResourceKeyboardSlotPicker.js

```diff
@@ -0,0 +1,311 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import injectT from '../../../../app/i18n/injectT';
+import { getIsSlotReserved, getIsSlotInPast, getInMs } from './resourceKeyboardSlotPickerUtils';
+
+const rootClass = 'app-ResourcePageKeyboardTimePicker';
+
+function getClassName(className) {
+  return `${rootClass}__${className}`;
+}
+
+const classes = {
+  root: rootClass,
+  fieldGrid: getClassName('field-grid'),
+};
+
+function getTime(dateTime) {
```

Comment:
This project uses moment so you could use moment().format()?

Reply:
That will also handle summer-winter time and timezone issues for you!

Reply:
I try to actively avoid using `moment` to make it easier to refactor out :D But you are right that here it's a borderline case--I wouldn't go through much more effort than this and not always even this much. I used moment on the parent component to format dates for instance.